app/assets/javascripts/discourse/lib/bootstrap-json/index.js

```diff
@@ -29,7 +29,7 @@ function htmlTag(buffer, bootstrap) {
   buffer.push(`<html lang="${bootstrap.html_lang}"${classList}>`);
 }
 
-function head(buffer, bootstrap) {
+function head(buffer, bootstrap, headers, baseURL) {
```

Comment:
Minor but the `headers` variable is kind of strange because it is never used in this function.

Reply:
Each builder has the same method signature and since it's not the last parameter I can't omit it since I need `baseURL`